python/test_file_generator.py

```diff
@@ -271,22 +278,22 @@ def s305_file(self):
 		utils.write_file(name=name, path=path, ts_input=ts, lar_input=lar)
 
 	def v611_file(self):
-		"""Sets loan type to 5 or blank."""
+		"""Sets loan type to 5."""
 		name = "v611.txt"
 		path = self.validity_path
 		ts = self.ts_df.copy()
 		lar = self.lar_df.copy()
-		lar.loan_type = lar.loan_type.map(lambda x: random.choice(("5", "")))
+		lar.loan_type = lar.loan_type.map(lambda x: random.choice(("5")))
 		print("writing {name}".format(name=name))
 		utils.write_file(name=name, path=path, ts_input=ts, lar_input=lar)
 
 	def v612_1_file(self):
-		"""Set loan purpose to 3 or blank."""
+		"""Set loan purpose to 3."""
 		name = "v612_1.txt"
 		path = self.validity_path
 		ts = self.ts_df.copy()
 		lar = self.lar_df.copy()
-		lar.loan_purpose = lar.loan_purpose.map(lambda x: random.choice(("3", "")))
+		lar.loan_purpose = lar.loan_purpose.map(lambda x: random.choice("3"))
```

Comment:
Removing blank allows the removal of the random.choice function. Recommend setting loan_purpose to 3.